db/migrations/2342234112_delete_old_forum_tables.sql

```diff
@@ -0,0 +1,56 @@
+-- +migrate Up
```

Comment:
The migration file name were corresponding to YYMMDDhhmm ... I suppose "42" is not the month, what is the logic?

Reply:
The logic was random ^^'. I edited the names and will change the few others I made in previous changes for consistency.